ibm_catalog.json

```diff
@@ -354,6 +354,9 @@
             {
               "key": "enable_jmx"
             },
+            {
+              "key": "agent_mode"
```

Comment:
use the new rich text widget here (see [example](https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/blob/608a4b0f81fd9b048a8c10d57964499a1fbb8e8a/ibm_catalog.json#L770-L779))